Assets/Plugins/Source/Editor/com.playeveryware.eos.editor.asmdef

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "com.playeveryware.eos-Editor",
+    "name": "com.playeveryware.eos.editor",
```

Comment:
That's interesting, I didn't realize that it would be a breaking change. What causes the breakage? Things seemed to function properly, so I'm curious where my preliminary testing is lacking.